---
uint8 isFind
uint16 detect_res
float conf